addons/account/models/ir_module.py

```diff
@@ -87,3 +87,10 @@ def _register_hook(self):
                 self.env.company,
             )
             del self.env.registry._auto_install_template
+
+    def module_uninstall(self):
+        unlinked_templates = [code for template in self.filtered('account_templates').mapped('account_templates') for code in template]
+        self.env['res.company'].search([
+                ('chart_template', 'in', unlinked_templates),
+                ]).chart_template = False
```

Comment:
3'd time is the charm, maybe?
```suggestion
        self.env['res.company'].search([
            ('chart_template', 'in', unlinked_templates),
        ]).chart_template = False
```

Reply:
Done